roles/init_dbserver/templates/postgresql.conf.template

```diff
@@ -3,7 +3,7 @@
 unix_socket_directories = '{{ pg_unix_socket_directories | join(",") }}'
 port = {{ pg_port }}
 listen_addresses = '*'
-wal_level = replica
+wal_level = logical
```

Comment:
I think this should be left as `replica` to not break the `setup_replication` role. Another template can be added with `logical` setting, or the `wal_level` can be added as a variable that needs to be validated against `supported_wal_level_values` and templated into this template.

Reply:
The setting "wal_level = logical" enables streaming replication, logical replication, and PGD. In other words, the "logical" level encompasses all replica usage scenarios, including logical decoding.

Reply:
As discussed. Resolving this conversation.